apps/storybook/src/Slider.stories.tsx

```diff
@@ -65,10 +82,19 @@ MultiThumbsAllowCrossing.args = {
 };
 
 export const WithCustomThumb: Story<SliderProps> = (args) => {
-  return <Slider {...args} />;
+  const thumbProps = args.thumbProps();
+  thumbProps.style = {
+    ...thumbProps.style,
+    ...(args.orientation === 'horizontal' // TODO: Is there a way to get the args inside WithCustomThumb.args? Because just to gets args, I have to move those lines of code from WithCustomThumb.args to WithCustomThumb
+      ? { transform: 'translateX(-19.2px)', top: 0 }
+      : { transform: 'translate(-25%, 50%)' }),
+  };
+  args.thumbProps = () => thumbProps;
+  return sliderWrapper(<Slider {...args} />, args);
 };
```

Comment:
I am calculating the `transform` and `top` properties that depend on the orientation. I am doing this in the `WithCustomThumb: Story<SliderProps> = ...` part. Is there a way to do this in the `WithCustomThumb.args` part? I don't think that's possible because I think `WithCustomThumb.args` can't change, but I wanted to confirm.

Reply:
**[DONE]** Yes, I felt so but just wanted to confirm, thanks